### ISSUE TITLE
VCAC-A Card build not working

### ISSUE THREAD

**Issue by goanto** [OpenVisualCloud/Smart-City-Sample#741]
Hi, recently we tried to deploy Smart City framework with VCAC-A card. In analytics container, we got an error related to vaapipostproc. We changed from Xeon to vcac-a by changing cmake to cmake -DPLATFORM=VCAC-A. It got reflected in the cmake process.
With VCAC-A build, we are getting error as below.
![Screenshot (162)](https://user-images.githubusercontent.com/27364117/107916254-fff02e80-6f8b-11eb-9ab6-7c1342e0ca8d.png)
We tried with Xeon in same environment it was working fine, screeshot for cmake -DPLATFORM=Xeon is below
![Screenshot (163)](https://user-images.githubusercontent.com/27364117/107916327-231ade00-6f8c-11eb-89d7-fd70a7abf493.png)
Kindly look into this issue

**Comment by nnshah2** [comment]
@goanto Can you confirm the settings with which the container are launched? It also seems as if the container may be running on the host system that doesn't have a GPU - can you confirm the container is targetting the VCAC-A card?

**Comment by nnshah1** [comment]
@goanto Can you confirm the settings with which the container are launched? It also seems as if the container may be running on the host system that doesn't have a GPU - can you confirm the container is targetting the VCAC-A card?

**Comment by goanto** [comment]
@nnshah1, I followed the smart city instruction for making platform with VCAC-A card. i Executed the below command 
Cmake -DPLATFORM=VCAC-A ..
It shows me that the platform is changed from Xeon to VCAC-A
We use a VCAC-A card and configured the Centos Kernel on top of it.
The images are created with make. The base image for analytics section is changed to smtc_analytics_common_vcac-a_gst.  I ran with traffic scenario, the base image for traffic scenario is changed to the above. For deployment i tried with Docker_Swarm and Kubernetes. Docker Swarm showed the above error.
